doc/container.md

```diff
@@ -266,6 +266,26 @@ with several versions of the Linux kernel, including at least kernel versions up
 If a kernel upgrade does not help, please use a different access mode for NFS-mounted directories,
 such as `--hidden-dir` or `--read-only-dir`.
 
+#### Problems due to default user namespaces restrictions on Ubuntu 24.04 (and possibly later versions)
+
+If you encounter any of the following errors/warnings while using container mode:
+
+- `Failed to configure container with operation 'fcntl.ioctl(sock, _SIOCSIFFLAGS, ifreq)': [Errno 1] Operation not permitted`
+-  `Failed to configure container with operation 'raise OSError(errno, msg)': [Errno 13] mount(None, b'/tmp/BenchExec_run_aiwscwd2', b'tmpfs', 0, b'size=100%') failed: Permission denied`
+- `Creating UID mapping into container failed: [Errno 1] Operation not permitted`
+- `Could not write to setgroups file in /proc: [Errno 13] Permission denied`
+- `Could not write to setgroups file in /proc: [Errno 13] Permission denied`
+- `Changing hostname in container prevented by system configuration, real hostname will leak into the container.`
+
+It seems that you are using BenchExec on Ubuntu 24.04 (or newer versions), which restricts the use of unprivileged user namespaces, making it impossible to change the hostname, create GID/UID mappings, or do some other operations to set up a container. You can temporarily disable AppArmor's restriction on unprivileged user namespaces by executing 
+```
+echo 0 | sudo tee /proc/sys/kernel/apparmor_restrict_unprivileged_userns
+```
+This setting will be reset upon reboot. To apply a persistent setting, create a new file at `/etc/sysctl.d/60-apparmor-namespace.conf` with the following contents:
+```
+kernel.apparmor_restrict_unprivileged_userns=0
+```
```

Comment:
We can move this to `INSTALL.md`, where we already explain how to do this for similar cases. And then just link to that part from here, like for the other problems.